src/otx/algo/utils/mmengine_utils.py

```diff
@@ -204,7 +204,7 @@ def load_checkpoint_to_model(
     """
     # get state_dict from checkpoint
     # TODO(Kirill): remove this when RTDETR weights is updloaded to openvino storage.
-    state_dict = checkpoint["ema"]["module"] if "ema" in checkpoint else checkpoint.get("state_dict", checkpoint)
+    state_dict = checkpoint["ema"]["module"] if "ema" in checkpoint else checkpoint.get("model_state", checkpoint)
```

Comment:
Doesn't it affect other tasks?

Reply:
Reverting this seems to pass all tests, and also confirms that the 3D ODM doesn't actually use this code, so I reverted it.
@kprokofi If there is a reason for this change, it would be nice to see it fixed in the next PR.